tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
But when you call ``$this->visit``, the ``isStarted`` method of the driver is still called. So your description of the interaction is incomplete (and btw, because it is incomplete, ``isStarted`` would return ``null``, not ``false``).

PHPUnit mocks don't complain about incomplete interaction specifications, which is why your test is passing (and a reason why I actually prefer Prophecy, but I haven't converted the whole Mink testsuite to it)